.github/workflows/tests.yml

```diff
@@ -71,16 +73,26 @@ jobs:
 
       - name: Start Selenium
         run: |
-          docker run --net host --name selenium --volume /dev/shm:/dev/shm --shm-size 2g selenium/standalone-firefox:2.53.1 &> ./logs/selenium.log &
+          docker run --net host --name selenium --volume /dev/shm:/dev/shm --shm-size 2g selenium/standalone-firefox:"${{ matrix.selenium }}" &> ./logs/selenium.log &
 
       - name: Wait for browser & PHP to start
         run: |
           while ! nc -z localhost 4444 </dev/null; do echo Waiting for remote driver to start...; sleep 1; done
           while ! nc -z localhost 8002 </dev/null; do echo Waiting for PHP server to start...; sleep 1; done
 
+      - name: Replace instaclick/php-webdriver for new selenium versions
+        run: |
+          if [ "${{ matrix.selenium }}" == "latest" ]; then
+            composer require lullabot/php-webdriver:dev-main
```

Comment:
This published fork does not publish the fact that it replaces `instaclick/php-webdriver` so composer will happily install both in parallel, with no guarantee about which one will win for defining the class. So this looks totally wrong to me.
In the current situation, I'm against merging this PR and releasing as is, because it actually makes the package unreliable for users.

Reply:
@stof it is marked as replacing it https://github.com/Lullabot/php-webdriver/blob/main/composer.json#L19 and I can see it doing so in the tests I just added:
https://github.com/minkphp/MinkSelenium2Driver/actions/runs/6392945083/job/17351217167?pr=372
```
Run if [ "latest" == "latest" ]; then
  if [ "latest" == "latest" ]; then
    composer require lullabot/php-webdriver:dev-main
  fi
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    COMPOSER_PROCESS_TIMEOUT: 0
    COMPOSER_NO_INTERACTION: 1
    COMPOSER_NO_AUDIT: 1
./composer.json has been updated
Running composer update lullabot/php-webdriver
Loading composer repositories with package information
Updating dependencies
Lock file operations: 1 install, 0 updates, 1 removal
  - Removing instaclick/php-webdriver (1.4.16)
  - Locking lullabot/php-webdriver (dev-main 3014d58)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 1 install, 0 updates, 1 removal
  - Downloading lullabot/php-webdriver (dev-main 3014d58)
  - Removing instaclick/php-webdriver (1.4.16)
  - Installing lullabot/php-webdriver (dev-main 3014d58): Extracting archive
Package phpunit/php-token-stream is abandoned, you should avoid using it. No replacement was suggested.
Generating autoload files
26 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
```

I'll make a proper release of the library before this new test is merged, I just wanted to get tests etc passing here first :smile:

Reply:
ah indeed. By default, packagist shows the info for the latest release, and this is only in the dev version.